test/resources/theme-ugly.css

```diff
@@ -95,4 +95,5 @@
   --sapField_TextColor: orange;
   --sapField_InformationBackground: #bc3f8c;
   --sapField_ReadOnly_Background: #4f9344;
+  --sapTextColor: white;
```

Comment:
Is it necessary?

Reply:
This is theme-ugly. It's for the test playground